packages/@lwc/integration-karma/test/component/dynamic-imports/x/dynamic/dynamic.js

```diff
@@ -17,10 +17,10 @@ export default class extends LightningElement {
     }
 
     @api enableCtor() {
-        this.loadCtor();
+        void this.loadCtor();
```

Comment:
What are these `void`s for?

Reply:
To satisfy [`no-floating-promises`](https://typescript-eslint.io/rules/no-floating-promises/). To avoid uncaught rejections, all promises should be appropriately handled (`return`, `await`, `.then(onResolved, onRejected)`, `.catch(onRejected)`). If we really truly want to allow uncaught rejections, we can use `void` to indicate that, rather than disabling the rule. In this case, I think allowing uncaught rejections is acceptable, because it's a test file. Presumably an uncaught rejection will cause the test to fail, which is the desired behavior.

Reply:
@wjhsf In that case, I'd prefer to add an`eslint` code comment to disable the check, since `void` is pretty mysterious otherwise.